examples/clic.md

```diff
@@ -0,0 +1,54 @@
+# Using the Key4hep-Stack for CLIC Simulation and Reconstruction
+
+
+This assumes that you have access to an installation of the Key4hep-stack, either via ``CVMFS`` or ``spack install``.
+
+These commands will explain how one can run the CLIC detector simulation and reconstruction using the Key4hep-Stack.
+First we will obtain all the necessary steering and input files for CLIC, simulate a few events and run the
+reconstruction both with ``Marlin`` and ``k4run``. These steps can be adapted to simulate or run other ``Marlin``
+processors as well.
+
+The ``CLICPerformance`` repository contains the steering and input files.
+```bash
+git clone https://github.com/iLCSoft/CLICPerformance
+cd CLICPerformance/clicConfig
+```
+
+## Simulation
+
+Now we can already simulate a few events
+```bash
+ddsim --compactFile $LCGEO/CLIC/compact/CLIC_o3_v14/CLIC_o3_v14.xml \
```

Comment:
This requires `$DD4HEP` to be set to the examples directory of the dd4hep installation. In spack this is not (yet) the case. There are really too many of these variables!

Reply:
Also, in the key4hep stack `dd4hep+lcio` is still missing, so this fails. I'll add that.

Reply:
Thanks for checking.
I have this diff in my local clicPerformance that fixes the problem you observe
```diff
-SIM.physics.pdgfile = os.path.join( os.environ.get("DD4HEP"), "DDG4/examples/particle.tbl" )
+SIM.physics.pdgfile = os.path.join( os.environ.get("DD4hepINSTALL"), "examples/DDG4/examples/particle.tbl" )
```

Reply:
I just tried with `source /cvmfs/clicdp.cern.ch/iLCSoft/lcg/97/nightly/x86_64-centos7-gcc8-opt/init_ilcsoft.sh` but then `$LCGEO` is not set. Would be great if it works with both stacks.

Reply:
set `lcgeo_DIR` from the spack recipe?
Doing something like
`${LCGEO-${lcgeo_DIR}}` probably just works in bash

Reply:
https://github.com/iLCSoft/CLICPerformance/pull/132 for the particle.tbl issue